cmake/libs/libfaiss.cmake

```diff
@@ -110,7 +110,7 @@ if(__X86_64)
             -Wno-unused-function
             -Wno-strict-aliasing>)
   target_link_libraries(
-    faiss PUBLIC OpenMP::OpenMP_CXX ${BLAS_LIBRARIES} ${LAPACK_LIBRARIES}
+    faiss PUBLIC OpenMP::OpenMP_CXX openblas ${BLAS_LIBRARIES} ${LAPACK_LIBRARIES}
```

Comment:
I'm not sure why we need `openblas` here, because there is already `${BLAS_LIBRARIES}` O_o